src/js/pages/universe/PackagesTab.js

```diff
@@ -30,6 +33,18 @@ const METHODS_TO_BIND = [
   'handleSearchStringChange'
 ];
 
+const shouldRenderUniverseOption = Hooks.applyFilter(
+  'hasCapability', false, 'packageAPI'
+);
+
+if (shouldRenderUniverseOption) {
```

Comment:
@jfurrow I'm going to approve for now just so we can get this in but I side with @philipnrmn that we should use a Mount and remove the option in a plugin. It's a little bizarre having this conditional in this file, especially in the JS module scope. Let's follow up on that JIRA :)